src/ResourceManager/Network/Commands.Network/Common/NetworkResourceManagerProfile.cs

```diff
@@ -137,6 +137,12 @@ protected override void Configure()
             Mapper.CreateMap<MNM.NetworkInterfaceDnsSettings, CNM.PSNetworkInterfaceDnsSettings>();
             Mapper.CreateMap<MNM.NetworkInterfaceIPConfiguration, CNM.PSNetworkInterfaceIPConfiguration>();
 
+            // Usage
```

Comment:
Autogenerated. Need to be copied and pasted manually. See [comment](https://github.com/EvgenyAgafonchikov/azure-powershell/commit/b810a9e0ef358449c00b313c9f3b224d2111178b#commitcomment-22086831)